tests/test_oclc.py

```diff
@@ -392,7 +392,7 @@ def test_extract_useful_data(self):
         eq_(1, len(subjects[Subject.FAST]))
         eq_(4, len(subjects[Subject.TAG]))
         eq_(1, len(subjects[Subject.PLACE]))
-        eq_(2, len(subjects[Subject.LCSH]))
+        eq_(3, len(subjects[Subject.LCSH]))
```

Comment:
Stick a comment here saying that http://subject.example.wo/internal_lookup didn't get included because we couldn't find any information about it.